packages/email/email.js

```diff
@@ -138,40 +137,39 @@ const getTransport = function() {
 };
 
 let nextDevModeMailId = 0;
-let output_stream = process.stdout;
 
-// Testing hooks
-EmailTest.overrideOutputStream = function(stream) {
-  nextDevModeMailId = 0;
-  output_stream = stream;
+EmailTest._getAndIncNextDevModeMailId = function () {
+  return nextDevModeMailId++;
 };
 
-EmailTest.restoreOutputStream = function() {
-  output_stream = process.stdout;
+// Testing hooks
+EmailTest.resetNextDevModeMailId = function () {
+  nextDevModeMailId = 0;
 };
 
-const devModeSend = function(mail) {
-  let devModeMailId = nextDevModeMailId++;
-
-  const stream = output_stream;
+const devModeSendAsync = async function (mail, stream) {
```

Comment:
There's no point in making this one `async` if it returns a `Promise` directly.

Reply:
You're right. Thanks!